nuxeo-core/nuxeo-core-query/src/main/java/org/nuxeo/ecm/core/query/sql/NXQL.java

```diff
@@ -18,13 +18,29 @@
  */
 package org.nuxeo.ecm.core.query.sql;
 
+import java.time.format.DateTimeParseException;
+
+import org.joda.time.DateTime;
+import org.joda.time.chrono.ISOChronology;
```

Comment:
Didn't we want to get rid of `joda.time`? See https://jira.nuxeo.com/browse/NXP-26517.

Reply:
Yes on master I'm replacing everything with `ZonedDateTime`

Reply:
See #3182